### PR TITLE
Added champions talk events

### PR DESCRIPTION
**Pull request by twinkarma**

### CODE REVIEW

File: _events/talk-2020-09-17-singularity-hpcchampions.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+published: true
+from: "15:40"
+to: "16:00"
+location: "Online"
+speaker: "Twin Karmakharm"
+slides_url: "https://docs.google.com/presentation/d/1Lwe94dIKxYH9XCK-iseQI2S9AuHWmmV7TXtzknmJ5do/edit?usp=sharing"
+title: "Singularity for HPC"
+---
+
+Twin Karmakharm gave a talk on Singualrity for HPC at the HPC Champions event. The talk gives an introduction to the Singularity container technology and experiences learned from its uses at the University of Sheffield.
```
Comment (willfurnass) [inline _events/talk-2020-09-17-singularity-hpcchampions.md]: /s/Singualrity/Singularity/

Comment (willfurnass) [reply _events/talk-2020-09-17-singularity-hpcchampions.md]: Hyperlink Singularity to https://sylabs.io/singularity/?

File: _events/talk-2020-09-17-jade-update-hpcchampions.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+category: seminar
+date: 2020-09-17
+published: true
+from: "13:50"
+to: "14:00"
+location: "Online"
+speaker: "Twin Karmakharm"
+slides_url: "https://docs.google.com/presentation/d/1kMiCRM_RrxlaZj_XUU3Ro5k5svY4pq8HrilHj9ck39w/edit?usp=sharing"
+title: "JADE/JADE 2 Updates"
```
Comment (willfurnass) [inline _events/talk-2020-09-17-jade-update-hpcchampions.md]: Maybe expand to "Updates on JADE and JADE 2 Tier 2 GPU-equipped HPC systems"?  Names may not be familiar to all readers.

**Review by willfurnass — CHANGES_REQUESTED** [review]

**Comment by willfurnass** [comment]
@twinkarma Suggested some minor edits; otherwise LGTM.  Cheers.

File: _events/talk-2020-09-17-singularity-hpcchampions.md (hunk 1 above)
Comment (twinkarma) [reply _events/talk-2020-09-17-singularity-hpcchampions.md]: Fixed in latest commit

File: _events/talk-2020-09-17-singularity-hpcchampions.md [hunk 3]
```diff
@@ -10,4 +10,4 @@ slides_url: "https://docs.google.com/presentation/d/1Lwe94dIKxYH9XCK-iseQI2S9AuH
 title: "Singularity for HPC"
 ---
 
-Twin Karmakharm gave a talk on Singualrity for HPC at the HPC Champions event. The talk gives an introduction to the Singularity container technology and experiences learned from its uses at the University of Sheffield.
+Twin Karmakharm gave a talk on [Singualrity](https://sylabs.io/singularity/) for HPC at the HPC Champions event. The talk gives an introduction to the Singularity container technology and experiences learned from its uses at the University of Sheffield.
```
Comment (willfurnass) [inline _events/talk-2020-09-17-singularity-hpcchampions.md]: Typo ('Singualrity')

**Review by willfurnass — CHANGES_REQUESTED** [review]

**Review by willfurnass — APPROVED** [review]

**Pull request merged**